public/assets/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
Why having multiple .gitignore files instead of one in root folder?

Reply:
I've always found it easier to manage what's ignored this way, but I'm not bothered either way.

Reply:
My problem is it is more difficult to maintain. Also do child files overwrite parent files? What if we miss .gitignore in one of the children - do we then control those children from parent?

Reply:
I am open to options but I never seen it in such setup hence need to know more.

Reply:
Would agree with Andrew - one is sufficient imo.

Reply:
Ok, let's get it all into one file 👍